app/assets/javascripts/application/bridge.js

```diff
@@ -0,0 +1,26 @@
+var mumuki = mumuki || {};
+(function (mumuki) {
+
+  function Laboratory(exerciseId){
+    this.exerciseId = exerciseId;
+  }
+
+  Laboratory.prototype = {
+    runTests: function (content) {
+      return $.ajax({
+        type: 'POST',
+        url: window.location + '/solutions',
+        data: {
+          solution: {
+            content: content
```

Comment:
We have to send content_metadata as well (the name is under revision :stuck_out_tongue:) or change the way the user and submittable solutions are saved